minio-operator.yaml

```diff
@@ -1,22 +1,22 @@
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
-  name: minioinstances.operator.min.io
+  name: tenant.minio.min.io
```

Comment:
I think we need this in `<plural>.<group>`, therefore `tenants.minio.min.io`
https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/